rosidl_typesupport_opensplice_cpp/resource/srv__type_support.cpp.template

```diff
@@ -105,8 +87,9 @@ public:
       return false;
     }
     if (status != DDS::RETCODE_OK) {
-      printf("take() failed. Status = %d\n", status);
-      throw std::runtime_error("Read failed");
+      std::stringstream ss;
```

Comment:
Including this line causes a compilation errors for the examples on Linux:

```
[ 91%] /home/jackie/code/ros2_ws/build/example_interfaces/rosidl_typesupport_opensplice_cpp/example_interfaces/srv/dds_opensplice/add_two_ints__type_support.cpp: In static member function ‘static bool rosidl_typesupport_opensplice_cpp::TemplateDataReader<example_interfaces::srv::dds_::Sample_AddTwoInts_Request_>::take_sample(DDS::DataReader*, rosidl_typesupport_opensplice_cpp::Sample<example_interfaces::srv::dds_::AddTwoInts_Request_>&)’:
/home/jackie/code/ros2_ws/build/example_interfaces/rosidl_typesupport_opensplice_cpp/example_interfaces/srv/dds_opensplice/add_two_ints__type_support.cpp:79:25: error: aggregate ‘std::stringstream ss’ has incomplete type and cannot be defined
       std::stringstream ss;
                         ^
Building CXX object CMakeFiles/example_interfaces__rosidl_typesupport_opensplice_cpp.dir/rosidl_typesupport_opensplice_cpp/example_interfaces/srv/dds_opensplice/Sample_AddTwoInts_Response_.cpp.o
/home/jackie/code/ros2_ws/build/example_interfaces/rosidl_typesupport_opensplice_cpp/example_interfaces/srv/dds_opensplice/add_two_ints__type_support.cpp: In static member function ‘static bool rosidl_typesupport_opensplice_cpp::TemplateDataReader<example_interfaces::srv::dds_::Sample_AddTwoInts_Response_>::take_sample(DDS::DataReader*, rosidl_typesupport_opensplice_cpp::Sample<example_interfaces::srv::dds_::AddTwoInts_Response_>&)’:
/home/jackie/code/ros2_ws/build/example_interfaces/rosidl_typesupport_opensplice_cpp/example_interfaces/srv/dds_opensplice/add_two_ints__type_support.cpp:165:25: error: aggregate ‘std::stringstream ss’ has incomplete type and cannot be defined
       std::stringstream ss;
```

`#include <sstream>` in this template fixes it.